Neo4j.Driver/Neo4j.Driver.Tests.TestBackend/Protocol/Result/Result.cs

```diff
@@ -55,6 +60,11 @@ public async Task PopulateRecords(IResultCursor cursor)
 			ResultCursor = cursor;
 			await Task.CompletedTask;
 		}
+
+		public async override Task<IResultSummary> ConsumeResults()
```

Comment:
As the two implementations are the same I think it would be cleaner to put the ConsumeResults into the base Result class as a public method and remove it from the two derived classes. Inheritance would take care of the rest.